jest.config.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-  verbose: true
+  verbose: true,
+  setupFilesAfterEnv: [
+    './test/setup.ts'
+  ]
```

Comment:
Seems your change from https://github.com/solid/solid-ui/pull/280 accidentally made it in here? Please remove.

Reply:
See https://github.com/solid/solid-ui/pull/283#discussion_r397188875